TAGradingServer/data/tables.sql

```diff
@@ -37,973 +38,597 @@ CREATE TABLE config (
-    ADD CONSTRAINT grades_questions_pkey PRIMARY KEY (grade_question_id);
-
-
---
--- Name: homework_grading_sections_pkey; Type: CONSTRAINT; Schema: public; Owner: -
---
-
-ALTER TABLE ONLY homework_grading_sections
-    ADD CONSTRAINT homework_grading_sections_pkey PRIMARY KEY (hgs_id);
-
-
---
--- Name: labs_pkey; Type: CONSTRAINT; Schema: public; Owner: -
---
-
-ALTER TABLE ONLY labs
-    ADD CONSTRAINT labs_pkey PRIMARY KEY (lab_id);
-
-
---
--- Name: late_days_pk; Type: CONSTRAINT; Schema: public; Owner: -
---
-
-ALTER TABLE ONLY late_days
-    ADD CONSTRAINT late_days_pk PRIMARY KEY (student_rcs, since_timestamp);
-
-
---
--- Name: pk_config; Type: CONSTRAINT; Schema: public; Owner: -
---
-
-ALTER TABLE ONLY config
-    ADD CONSTRAINT pk_config PRIMARY KEY (config_name);
-
-
---
--- Name: pkey_ex_id; Type: CONSTRAINT; Schema: public; Owner: -
---
-
-ALTER TABLE ONLY late_day_exceptions
-    ADD CONSTRAINT pkey_ex_id PRIMARY KEY (ex_id);
-
-
---
--- Name: questions_pkey; Type: CONSTRAINT; Schema: public; Owner: -
---
+CREATE TABLE users (
+    user_id character varying NOT NULL,
+    user_firstname character varying NOT NULL,
+    user_lastname character varying NOT NULL,
+    user_email character varying NOT NULL,
+    user_group integer NOT NULL,
+    registration_section integer,
+    rotating_section integer,
+    CONSTRAINT users_user_group_check CHECK (((user_group >= 0) AND (user_group <= 4)))
+);
 
-ALTER TABLE ONLY questions
-    ADD CONSTRAINT questions_pkey PRIMARY KEY (question_id);
 
 --
--- TOC entry 2369 (class 2606 OID 25661)
--- Name: pkey_grades_other_id; Type: CONSTRAINT; Schema: public; Owner: -
+-- Name: gc_id; Type: DEFAULT; Schema: public; Owner: -
 --
 
-ALTER TABLE ONLY grades_others
-ADD CONSTRAINT pkey_grades_other_id PRIMARY KEY (grades_other_id);
+ALTER TABLE ONLY gradeable_component ALTER COLUMN gc_id SET DEFAULT nextval('gradeable_component_gc_id_seq'::regclass);
 
 
 --
--- TOC entry 2365 (class 2606 OID 25646)
--- Name: pkey_other_grades_id; Type: CONSTRAINT; Schema: public; Owner: -
+-- Name: gd_id; Type: DEFAULT; Schema: public; Owner: -
 --
 
-ALTER TABLE ONLY other_grades
-ADD CONSTRAINT pkey_other_grades_id PRIMARY KEY (oid);
-
---
--- Name: relationships_users_pkey; Type: CONSTRAINT; Schema: public; Owner: -
---
-
-ALTER TABLE ONLY relationships_users
-    ADD CONSTRAINT relationships_users_pkey PRIMARY KEY (relationship_user_id);
+ALTER TABLE ONLY gradeable_data ALTER COLUMN gd_id SET DEFAULT nextval('gradeable_data_gd_id_seq'::regclass);
 
 
 --
--- Name: rubrics_name_unique; Type: CONSTRAINT; Schema: public; Owner: -
+-- Data for Name: electronic_gradeable; Type: TABLE DATA; Schema: public; Owner: -
 --
 
-ALTER TABLE ONLY rubrics
-    ADD CONSTRAINT rubrics_name_unique UNIQUE (rubric_name);
+COPY electronic_gradeable (g_id, eg_instructions_url, eg_submission_open_date, eg_submission_due_date, eg_is_repository, eg_subdirectory, eg_use_ta_grading, eg_config_path) FROM stdin;
```

Comment:
You need to re-export this SQL file and not have it do anything with the data and not have these COPY statements.